testutil/keeper/session.go

```diff
@@ -216,6 +218,17 @@ func defaultSupplierKeeperMock(t testing.TB) types.SupplierKeeper {
 	return mockSupplierKeeper
 }
 
+func defaultSharedKeeperMock(t testing.TB) types.SharedKeeper {
```

Comment:
sharedKeeperForDefaultParamsMock

Reply:
"Default" in the function name here is unrelated to the default shared params. The naming here is for consistency within this pkgs helpers. I don't think the name needs to encode any semantic meaning about the behavior of this mock; otherwise, I would feel obligated to rename the other `default...Mock()` functions.

I think the renaming of the other helper methods (`...WithDefaultParams()`) which do involve default shared params sufficiently disambiguates things.